### ISSUE TITLE
[Review] Questions on submission review process

### ISSUE THREAD

**Issue by jungshadow** [it-dept-cis/RABET-V-Pilot#8]
We received one comment that was somewhat open-ended on the [Submission Review Process](https://github.com/it-dept-cis/RABET-V-Pilot/blob/master/docs/source/Activities/Submission_Review_Process.md).

> Do you plan to have more to say about each of these? Or is that something the pilot will flesh out, as we figure out the right details for each workflow?
>
>Especially if people will need to be trained to do this workflow, this subsection could become its own paper, for better or worse. 
>
> I don’t think any parts are missing; in that sense, it’s complete. 

In some ways, it seems the comment resolved itself, but I figured I raise it here to see if anyone thought we should modify anything, either in the narrative or the content itself.

**Comment by wilsoaa** [comment]
I agree with the sentiment. The process sections are NOT complete. We will have much more detail to add as we go through and eventually conclude the pilot.
